docs/delorean/configuring-jenkins.md

```diff
@@ -0,0 +1,72 @@
+# Configuring Jenkins
+
+## Table of Contents
+  - [1. Prerequisites](#1-prerequisites)
+  - [2. Configuring the inventory](#2-configuring-the-inventory)
+    - [2.1 Host File](#host-file)
+    - [2.2 Host Vars](#host-vars)
+    - [2.3 Credentials and Global Settings](#credentials-and-global-settings)
+  - [3. Running the Script](#4-running-the-script)
+    - [3.1 Plugins Installation](#plugins-installation)
+    - [3.2 Potential Issues During Configuraiton](#potential-issues-during-configuration)
+
+## Prerequisites
+- [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#intro-installation-guide)
+- [Docker](https://docs.docker.com/install/overview)
+
+
+The `configure_jenkins` playbook and all of its required files for inventory and roles are located in the `scripts/` folder.
+
+```sh
+cd scripts/
+```
+## Configuring the inventory
+The inventory files must be configured before the script can be run.
+
+### Host File
+Create the `hosts` file
+```sh
+cp inventories/hosts.template inventories/hosts
+```
+
+### Host Vars
+A host vars file must be created for each Jenkins host. This will contain the Jenkins configuration (i.e. credentials) used to connect to the target Jenkins instance.
+```sh
+cp inventories/host_vars/jenkins-host.template inventories/host_vars/<jenkins-host>.yaml
+```
```

Comment:
It's important that the jenkins host entry under the "jenkins" group in the hosts file has the same name as this host vars file.

```
[all:vars]
ansible_connection=local

[jenkins]
<jenkins-host>
```
```
inventories/host_vars/<jenkins-host>.yaml
```

Might be wroth calling that out.